manifests/exporter.pp

```diff
@@ -7,7 +7,7 @@
 
 class rsan::exporter (
   $rsan_host = undef,
-  String $rsanip = rsan::get_rsan_ip(),
+  Array $rsanip = rsan::get_rsan_ip(),
```

Comment:
Now that this is an array I'd recommend considering changing the naming e.g.

* Things that return an array should be plural
* `rsan::get_rsan_ip` actually technically gets the IPs of all "importers", not all RSAN nodes (i.e. not exporters) so possibly the naming could be improved here
* `$rsanip` in a way variable doesn't contain the RSAN IP, or at least that's not how it's used. What it really contains is a list of IPs that are allowed to mount the exported directories. Maybe we could make the name of this more clearly articulate that

Reply:
Great point. Updated